src/patterns/task-list-pages/index.md.njk

```diff
@@ -51,7 +53,9 @@ Where possible, task names should:
 
 ### Show the status of the tasks
 
-Include a summary above the task list to say how many tasks or sections have been completed.
+Include a summary above the task list to say how many tasks have been completed.
+
+You might find a better word to describe what a ‘task’ is for your users. If your service is an application form, each task might a ‘section’ of questions to complete.
```

Comment:
Looks like there's a word missing here, cc @calvin-lau-sig7

Reply:
@vanitabarrett well spotted! I've added the missing word "be" in b04446ca8aebb966e3d90f9366dad6869ceba160.